src/modules/products/services/product.service.ts

```
@@ -0,0 +1,77 @@
+@Injectable()
+export class ProductService {
+  constructor(
+    @InjectRepository(Product)
+    private readonly productRepository: Repository<Product>,
+  ) {}
+
+  async create(createProductDTO: CreateProductDTO): Promise<Product> {
+    try {
+      const product = this.productRepository.create(createProductDTO);
+      return await this.productRepository.save(product);
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)